client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -433,9 +436,16 @@ static void addSearchRequestParams(Params params, SearchRequest searchRequest) {
         params.putParam(RestSearchAction.TYPED_KEYS_PARAM, "true");
         params.withRouting(searchRequest.routing());
         params.withPreference(searchRequest.preference());
+        if (searchRequest.pointInTimeBuilder() == null) {
+            params.withIndicesOptions(searchRequest.indicesOptions());
```

Comment:
It seems to be not needed, the next line does `params.withIndicesOptions(searchRequest.indicesOptions());` unconditionally